cron/main.go

```diff
@@ -0,0 +1,42 @@
+package cron
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+const (
+	host	= "trotter.cyverse.org"
+	port   	= 6432
+	user   	= "de"
+	dbname 	= "de"
+	folder  = "/tmp"
+	logfile = "logs-stdout-output"
+)
+
+
+func InitDB() *sql.DB{
+	psqlInfo := fmt.Sprintf("host=%s port=%d user=%s "+
+		"dbname=%s sslmode=disable", host, port, user, dbname)
+	db, err := sql.Open("postgres", psqlInfo)
+
+	if err != nil {
+		panic(err)
+	}
+
+	err = db.Ping()
+	if err != nil {
+		panic(err)
+	}
+
+	return db
+}
+
+
+func main(){
+	db := InitDB()
+	defer db.Close()
+	amount := 10
+	days := 100
+	GetTopApps(db, amount, days)
+}
```

Comment:
I think that the cron job is going away completely. You can probably delete this code.

Reply:
This code returns a db object that the other packages use to run queries on.

Reply:
Oh, I was primarily thinking about the `main` function. I hadn't realized that `InitDB` was being called from elsewhere.